Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -214,14 +214,15 @@ public function reload() : bool
 
     public function getFirstPostId() : int
```

Comment:
Maybe we should rename it to `getFirstVisiblePostId` (see comment below).

Reply:
we could add a optional parameter for both methods ($exclude_root_node = false)